### PR TITLE
fix(ui5-radio-button): SAP Horizon theme updates

### PR DESCRIPTION
**Pull request by dimovpetar**
- increase distance of focus border in Horizon themes
- delete box-shadow styles in Horizon themes
- make the test page responsive
- delete unused CSS variables

### CODE REVIEW

File: packages/main/src/themes/RadioButton.css [hunk 1]
```diff
@@ -100,8 +84,12 @@
...
 /* with label */
-:host([text]) .ui5-radio-root:focus:before {
-	inset-inline: var(--_ui5_radio_button_focus_dist) 0;
+:host([text]) .ui5-radio-root {
+	padding-right: var(--_ui5_radio_button_border_width);
+}
+
+:host([text]) .ui5-radio-root:focus::before {
+	right: 0px;
 }
```
Comment (gmkv) [inline packages/main/src/themes/RadioButton.css:93]: use the logical properties for this css

**Review by gmkv — CHANGES_REQUESTED** [review]

File: packages/main/src/themes/RadioButton.css (hunk 1 above)
Comment (dimovpetar) [reply packages/main/src/themes/RadioButton.css]: Done

File: packages/main/src/themes/RadioButton.css [hunk 2]
```diff
@@ -42,44 +41,29 @@
...
+:host(:not([disabled])) .ui5-radio-root:focus::before {
 	content: "";
 	display: var(--_ui5_radio_button_focus_outline);
 	position: absolute;
 	top: var(--_ui5_radio_button_focus_dist);
 	bottom: var(--_ui5_radio_button_focus_dist);
-	left: var(--_ui5_radio_button_focus_dist);
-	right: var(--_ui5_radio_button_focus_dist);
+	inset-inline-start: var(--_ui5_radio_button_focus_dist);
+	inset-inline-end: var(--_ui5_radio_button_focus_dist);
```
Comment (gmkv) [inline packages/main/src/themes/RadioButton.css:51]: all these can be rewritten with just `inset` since the distance is the same on every side

**Review by gmkv — CHANGES_REQUESTED** [review]

File: packages/main/src/themes/RadioButton.css (hunk 2 above)
Comment (dimovpetar) [reply packages/main/src/themes/RadioButton.css:51]: Done

**Review by gmkv — APPROVED** [review]

**Pull request merged**